environment.yml

```diff
@@ -1,28 +1,36 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-sarek-2.5.2
+name: nf-core-sarek-2.6
 channels:
   - conda-forge
   - bioconda
   - defaults
 dependencies:
-  - ascat=2.5.2
-  - bcftools=1.9
-  - bwa=0.7.17
-  - cancerit-allelecount=4.0.2
-  - control-freec=11.4
-  - ensembl-vep=95.2
-  - fastqc=0.11.8
-  - freebayes=1.2.0
-  - gatk4=4.1.2.0
-  - genesplicer=1.0
-  - htslib=1.9
-  - manta=1.5.0
-  - multiqc=1.7
-  - qualimap=2.2.2b
-  - samtools=1.9
-  - snpeff=4.3.1t
-  - strelka=2.9.10
-  - tiddit=2.7.1
-  - vcfanno=0.3.1
-  - vcftools=0.1.16
\ No newline at end of file
+  - conda-forge::markdown=3.1.1
+  - conda-forge::pymdown-extensions=6.0
+  - conda-forge::pygments=2.5.2
+  - bioconda::ascat=2.5.2
+  - bioconda::bcftools=1.9
+  - bioconda::bwa=0.7.17
+  - bioconda::cancerit-allelecount=4.0.2
+  - bioconda::cnvkit=0.9.6
+  - bioconda::control-freec=11.5
+  - bioconda::ensembl-vep=99.2
+  - bioconda::fastqc=0.11.9
+  - bioconda::freebayes=1.3.2
+  - bioconda::gatk4-spark=4.1.6.0
+  - bioconda::genesplicer=1.0
+  - bioconda::htslib=1.9
+  - bioconda::manta=1.6.0
+  - bioconda::msisensor=0.5
+  - bioconda::multiqc=1.8
+  - bioconda::qualimap=2.2.2d
+  - bioconda::samtools=1.9
+  - bioconda::snpeff=4.3.1t
+  - bioconda::strelka=2.9.10
+  - bioconda::tiddit=2.7.1
+  - bioconda::trim-galore=0.6.5
+  - bioconda::vcfanno=0.3.2
```

Comment:
Missing from scrape_software_versions, if important.

Reply:
Not useful